package.json

```diff
@@ -14,7 +14,7 @@
     "d3-axis": "1.0.x",
     "d3-scale": "1.0.x",
     "d3-selection": "1.3.x",
-    "eslint": "7.0.0",
+    "eslint": "^7.0.0",
```

Comment:
Let's lock it to a specific version. We moved away from version ranges because not everyone uses semver and breaking changes were producing unreliable builds. And because package-lock sucks.